### PR TITLE
fix(DB/gossip_menu_option_locale): Populate missing locale entries from broadcast_text (zhCN)

### PR DESCRIPTION
**Pull request by ZhengPeiRu21**
## Changes Proposed:
-  Use existing broadcast_text_locale data to populate missing gossip_menu_option_locale entries.
-  I only populated missing entries; in the case of conflicts I did not change anything
- This is the zhCN locale entries - this is split into multiple PR to prevent too large SQL files

## Issues Addressed:

- Closes unreported issue

## SOURCE:

broadcast_text_locale data

## Tests Performed:

- N/A

## How to Test the Changes:

1. Use not enUS client and see gossip menu options now have before missing locale data

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by ZhengPeiRu21** [comment]
Sorry, I am just now realize I should first have a SQL DELETE to prevent conflict. For this large type of bulk insert, we are prefer many individual DELETEs, or just use INSERT IGNORE or REPLACE INTO?

**Comment by mpfans** [comment]
发现缺少一些zhCN的翻译,如6185\6186\6187-0 等等

另外我还建议使用REPLACE INTO,然后把zhCN和zhTW单独出来,其他的语言再单独提交一个PR...

上次提交的PR因为文件过大,自动更新时出现了失败的情况.

**Comment by acidmanifesto** [comment]
What is the status of this PR?

**Comment by ZhengPeiRu21** [comment]
> Do you still need to test

Thank you for willing to test! I am still needing to split this PR into multiple PR to avoid the too large SQL file, but it will still be the same updates, so it can be tested as-is. Please let me know if you see any issue!

**Comment by ZhengPeiRu21** [comment]
I have updated this PR to split each language into separate PR in order to prevent large SQL files. This PR now contains only zhCN locale data.

**Comment by ZhengPeiRu21** [comment]
I think this PR should be ready to merge. WIP tag can be remove.

**Comment by Nyeriah** [comment]
Hmm SQL import is erroring

**Comment by yuanf225** [comment]
image
I just tested it and the import is fine

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1654272234519507300.sql [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+(10704, 1, 'zhCN', '我还没有完成十字军的试炼。'),
+(10704, 2, 'zhCN', '我们要向这些野兽再发出狠狠的一击！'),
+(10706, 0, 'zhCN', '是的，我们已经准备好了迎接即将到来的挑战。'),
+(10847, 0, 'zhCN', '开始战斗'),
+(10854, 0, 'zhCN', '奥术师泰拜林说你会借我一件战袍。'),
+(10854, 1, 'zhCN', '魔导师哈瑟尔说你会借我一件战袍。'),
+(10933, 0, 'zhCN', '我们准备好了，穆拉丁。'),
+(10934, 0, 'zhCN', '确信无疑。出发吧！'),
+(10943, 0, 'zhCN', '有什么我能效劳的，我的女士？'),
+(10952, 0, 'zhCN', 'Lok''tar ogar！我们准备好了！兽人兄弟们，冲啊！'),
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1654272234519507300.sql:398]: this line is erroring out the check with the ''
```suggestion
(10952, 0, 'zhCN', 'Lok tar ogar！我们准备好了！兽人兄弟们，冲啊！'),
```

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Comment by acidmanifesto** [comment]
@ZhengPeiRu21 i have not forgot about this pr, however i am starting to think its because it is entirely too big with the replace is why it fails. I am going to try to split up the query and see how that goes.

**Review by acidmanifesto — APPROVED** [review]
I did alot of lines

**Pull request merged**